userservice/src/main/java/com/gymbro/userservice/controller/UserController.java

```diff
@@ -0,0 +1,25 @@
+package com.gymbro.userservice.controller;
+
+import com.gymbro.userservice.DTO.UserDto;
+import com.gymbro.userservice.service.UserService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@AllArgsConstructor
+@RequestMapping("/users")
```

Comment:
please add **/api** before /users to preserve consistency across the API

Reply:
nah